pmm/pmm-2-0-documentation-md.groovy

```diff
@@ -0,0 +1,79 @@
+pipeline {
+    agent {
+        label 'micro-amazon'
+    }
+    parameters {
+        string(
+            defaultValue: 'master',
+            description: 'Tag/Branch for build',
+            name: 'BRANCH_NAME')
+        choice(
+            choices: ['test.percona.com', 'percona.com'],
+            description: 'Publish to test or live website',
+            name: 'PUBLISH_TARGET')
+    }
+    options {
+        buildDiscarder(logRotator(daysToKeepStr: '14', numToKeepStr: '5'))
+    }
+    stages {
+        stage('Checkout') {
+            steps {
+                sh '''
+                    sudo chmod 777 -R ./
+                '''
+                git poll: false, branch: BRANCH_NAME, url: 'https://github.com/percona/pmm-doc.git'
+            }
+        }
+        stage('Doc Prepare') {
+            steps {
+                withCredentials([sshUserPrivateKey(credentialsId: 'publish-doc-percona.com', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                    sh '''
+                        sudo yum -y install docker
+                        sudo usermod -aG docker ec2-user
+                        sudo service docker start
+                        sg docker -c "
+                            docker pull perconalab/pmm-doc-md:latest
+                            docker run -i -v `pwd`:/docs -e USER_ID=$UID -e UMASK=0777 perconalab/pmm-doc-md
```

Comment:
Are you sure that UMASK should be 777? According to documentation, umask is reversed, so umask 777 means is to disallow all in read&write&execute

Reply:
@puneet0191 can answer. Seems to be a standard workaround for a common failure.

Reply:
I missed it, I don't recall the context, but I used it this with slow log to avoid changing permission, not sure what is the context here.